pyspeckit/spectrum/models/model.py

```diff
@@ -307,18 +319,6 @@ def f(p,fjac=None):
                 return [0,residuals]
         return f
 
-    def __call__(self, *args, **kwargs):
```

Comment:
this is just moved this below the __init__ method, and in the multisingle branch the check for multisingle is gone, should I rebase on top of it?